ddsrouter_core/src/cpp/participant/implementations/auxiliar/EmptyParticipant.cpp

```diff
@@ -13,54 +13,54 @@
 // limitations under the License.
 
 /**
- * @file VoidParticipant.cpp
+ * @file EmptyParticipant.cpp
```

Comment:
Why did you decide to change this name?
For me `empty` means that there is nothing inside, while `void` means it does nothing.

@juanlofer-eprosima opinion?

Reply:
I'd keep `void`

Reply:
It was explained in the PR description. The casting from previous enum value `VOID` into `void` (enum values should be lowercase to distinguish them from macros see https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#enum5-dont-use-all_caps-for-enumerators), make it clash with the `void` type. So I chose `empty`, but we could use other like  `blank`, `null`, `nullified`, `naught`.... Let me know